db/migrate/20220714153256_create_comments.rb

```diff
@@ -0,0 +1,10 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.string :text
```

Comment:
- [ ] Text type should be text not string. Kindly change it.

Reply:
- [ ] as the previous CR already mentioned please change the type for text to Text not string.